src/OmniSharp.MSBuild/ProjectLoadListener.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Execution;
+using Microsoft.Extensions.Logging;
+using OmniSharp.Eventing;
+using OmniSharp.Models;
+using OmniSharp.MSBuild.Notification;
+
+namespace OmniSharp.MSBuild
+{
+    [Export(typeof(IMSBuildEventSink))]
+    public class ProjectLoadListener : IMSBuildEventSink
+    {
+        internal const string TargetFrameworkPropertyName = "TargetFramework";
```

Comment:
style points, you can do `internal const string TargetFramework = nameof(TargetFramework)`